readme.md

```diff
@@ -18,9 +18,15 @@ $ latest-version --help
   Usage
     $ latest-version <package-name>
 
+  Options
+    --range    Specify which semver range to use to find the latest version
```

Comment:
This is inconsistent with the actual output. Please just copy-paste the output to get it correctly.

Reply:
Good point, thanks!

Updated.